templates/config.hocon.tmpl

```diff
@@ -77,6 +77,13 @@ collector {
       timeLimit = ${time_limit_ms}
     }
   }
+  telemetry {
```

Comment:
Doesn't this need a collector endpoint for the telemetry?

Reply:
Updated to use the same url as the telemetry module